src/views/v2/commerce/order/details/total.php

```diff
@@ -31,6 +31,6 @@
 		<?php esc_html_e( 'Total:', 'event-tickets' ); ?>
 	</div>
 	<div class="tribe-tickets__commerce-order-details-col2">
-		<?php echo esc_html( $order->formatted_total ); ?>
+		<?php echo esc_html( \TEC\Tickets\Commerce\Utils\Price::to_currency( $order->formatted_total ) ); ?>
```

Comment:
We need to set the @version of this template TBD and add a @since TBD with a small description of the changes.